### DISCUSSION TITLE
meeting_0711

### DISCUSSION THREAD

**Discussion by gglin001** [PFCCLab/paddlefx#61] [General]
# meeting minutes

- 需要支持支持的模型

  - resnet
  - BERT
  - 带有控制流的模型, 自己构建一个就好了

- 最近的目标: 支持 `examples/native_interpreter` 使用 dynamo 实现 e2e 的运行

- 在 dynamo trace 没有 ready 的情况下, 可以使用 symbolic_trace 生成的 Graph 进行测试
- 变量追踪机制, 可以使用 paddlesot 的部分, 这部分由 @zrr1999 在后续添加
- 需要支持将 paddlefx.Graph 转换到 ONNX 实现模型的运行, 这部分将由 @sanbuphy 在后续添加
- dynamo 部分, 需要支持 graph break 和 inline Translator, 这部分由 @gglin001 在后续添加
- other misc

## ref

onnx 相关
torch-trt https://github.com/pytorch/TensorRT
onnxscript https://github.com/microsoft/onnxscript

**Comment by sanbuphy** [comment]
onnx graphsurgeon 可以制造onnx图， onnx原生api可以导入权重信息
